R/eval-select.R

```diff
@@ -36,6 +36,9 @@
 #' @param allow_rename If `TRUE` (the default), the renaming syntax
 #'   `c(foo = bar)` is allowed. If `FALSE`, it causes an error. This
 #'   is useful to implement purely selective behaviour.
+#' @param allow_empty If `TRUE` (default), its ok for `expr` to result
+#'   in no variables. If `FALSE`, will error if `expr` yields an empty
+#'   selection.
```

Comment:
```suggestion
#' @param allow_empty If `TRUE` (the default), it is ok for `expr` to result
#'   in an empty selection. If `FALSE`, will error if `expr` yields an empty
#'   selection.
```

Just trying to be consistent with `allow_rename`